addons/flannel/0.21.5/install.sh

```diff
@@ -295,6 +317,10 @@ function flannel_scale_up_rook() {
 function weave_to_flannel() {
     local dst="$DIR/kustomize/flannel"
 
+    # start by creating a config map indicating that this migration has been started. if we fail
+    # midair this ensure this function will be ran once more.
+    kubectl create configmap -n kube-system migrating-from-weave --from-literal=started="true" --dry-run=client -o yaml | kubectl apply -f -
```

Comment:
Currently, we have already `kurl-migration-from-rook` see https://github.com/replicatedhq/kURL/blob/a99174144c2aa8a26717b48709ea975cddf44456/scripts/common/rook.sh#L419

Could we follow the same standard and keep the configmap in the kurl namespace since it is a kurl thing?

Reply:
Done.